### PR TITLE
Add fl: a new binary package.

### PR DESCRIPTION
**Pull request by weijianwen**

### CODE REVIEW

File: var/spack/repos/builtin/packages/fl/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class Fl(Package):
+    """Fawlty Language is an IDL8 (Interactive Data Language) compatible compiler."""
+
+    homepage = "https://www.flxpert.hu/fl/"
+    url      = "https://www.flxpert.hu/fl/fl_0.79.47-amd64-linux.tar.gz"
+
+    version('0.79.47', sha256='b8a4a74118c1a024313bf912261fbc016a53f2d15adb1226217f2a10a9f7ca9a')
+
+    # TODO: Add runtime dependency since Fawlty is binary distributed.
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/fl/package.py]: Does this mean the `fl` package depends on something to run?

Is there something preventing this being added now?

File: var/spack/repos/builtin/packages/fl/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fl(Package):
+    """Fawlty Language is an IDL8 (Interactive Data Language) compatible compiler."""
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/fl/package.py]: @weijianwen flake is complaining about this line length

File: var/spack/repos/builtin/packages/fl/package.py (hunk 1 above)
Comment (weijianwen) [reply var/spack/repos/builtin/packages/fl/package.py]: I've successfully lanched the installed `fl64_gui` app on a X86 CentOS server. TODO will be safely removed.

**Comment by weijianwen** [comment]
ping

**Pull request merged**